### ISSUE TITLE
Add new game options

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#873]
Add new game options when starting a game.
Should have at least planet selection.
And a LAWK switch (if thermoplast gets added).

This should hook into: https://github.com/Revolutionary-Games/Thrive/issues/872
Which should then affect the patch map generation.

The setup screen that allows to set the seed for the procedural patch map so that the seeds can be shared. Once this is done: https://github.com/Revolutionary-Games/Thrive/pull/3278

**Comment by hhyyrylainen** [comment]
As for the graphics a basic "terran planet" icon is what I can think of this needing that we don't have yet. And maybe also a "copy to clipboard" icon.

**Comment by hhyyrylainen** [comment]
Dev forum discussion: https://forum.revolutionarygamesstudio.com/t/implementing-new-game-settings/884

**Comment by hhyyrylainen** [comment]
Split the graphics tasks into a separate issue: https://github.com/Revolutionary-Games/Thrive/issues/3425
